src/test/ui/parser/assoc-type-in-type-arg.stderr

```diff
@@ -0,0 +1,8 @@
+error: associated type bounds do not belong here
```

Comment:
I'd rephrase this as "bounds on associated types" because the `associated_type_bounds` lang feature already uses this terminology and it's not exactly the same.